### PR TITLE
Add Natural product, NMR data and Pharmacology, ADME, metabolism sect…

### PR DESCRIPTION
**Pull request by phalem**
some of ADME and metabolism links inspired from Assoc. Prof. Johannes Kirchmair lecture

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Contributions are very welcome - please follow the [guidelines](CONTRIBUTING.md)
 - [nCov-Group Data Repository](https://2019-ncovgroup.github.io/data/#dataset-downloads): SMILES, fingerprints, descriptors, and images of millions of compounds.
 - [zinc20](https://files.docking.org/zinc20-ML/): ZINC20 library prepared for Deep Docking-accelerated virtual screening
 - [zinc22](https://cartblanche22.docking.org/): commercially-available compounds for virtual screening
+- [COCONUT](https://coconut.naturalproducts.net/): is an open source project for Natural Products (NPs) storage, search and analysis.
+- [nmrshiftdb2](https://nmrshiftdb.nmr.uni-koeln.de/nmrshiftdb/media-type/html/user/anon/page/default.psml/js_pane/P-Help;jsessionid=8DC5FC88F23A161E2276E00C85C184CA): is a NMR database (web database) for organic structures and their nuclear magnetic resonance (nmr) spectra.
```
Comment (kjappelbaum) [inline README.md:29]:
```suggestion
- [nmrshiftdb2](https://nmrshiftdb.nmr.uni-koeln.de/A): is database for organic structures and their nuclear magnetic resonance (NMR) spectra. 
```

File: README.md [hunk 2]
```diff
@@ -51,11 +54,20 @@ Contributions are very welcome - please follow the [guidelines](CONTRIBUTING.md)
...
 ## Target identification data
-- [Open Targets](https://platform.opentargets.org/): is a large-scale resource that uses human genetics and genomics data for systematic drug target identification and prioritization.
 
+- [Open Targets](https://platform.opentargets.org/): is a large-scale resource that uses human genetics and genomics data for systematic drug target identification and prioritization.
 - [Probes & Drugs Portal](https://www.probes-drugs.org/data_sources): is an interactive, open data resource for chemical biology. Overview of libraries of bioactive compounds (e.g., ChEMBL, Guide to PHARMACOLOGY), including commercial screening libraries.
 
+## Pharmacology & ADME & Metabolism
+
 - [Guide to PHARMACOLOGY](https://www.guidetopharmacology.org/): is an expert-curated resource of ligand-activity-target relationships. It includes activity data even for data with unknown bioactivity value (under CC BY-SA 4.0).
+- [Drug Indications Database(DID)](https://doi.org/10.1186/s13326-016-0110-0): is a data of structured drug-indication relations. It is intended to facilitate building practical, comprehensive, integrated drug ontologies.
```
Comment (kjappelbaum) [inline README.md:64]:
```suggestion
- [Drug Indications Database (DID)](https://figshare.com/articles/dataset/Additional_file_1_of_Toward_a_comprehensive_drug_ontology_extraction_of_drug-indication_relations_from_diverse_information_sources/4535021): is a dataset of structured drug-indication relations. It is intended to facilitate the building of practical, comprehensive, integrated drug ontologies.
```

**Review by kjappelbaum — COMMENTED** [review]
Thanks a lot for all your contributions 👍🏽 
Very appreciated 💯

File: README.md [hunk 3]
```diff
@@ -51,11 +54,20 @@ Contributions are very welcome - please follow the [guidelines](CONTRIBUTING.md)
...
+- [The Metabolism and Transport Database ](https://www-metrabase.ch.cam.ac.uk/metrabaseui/pageview/download/):  is a cheminformatics and bioinformatics resource that contains curated data related to human small molecule metabolism and transport.
```
Comment (kjappelbaum) [inline README.md:65]:
```suggestion
- [The Metabolism and Transport Database ](https://www-metrabase.ch.cam.ac.uk/metrabaseui/pageview/download/): is a cheminformatics and bioinformatics resource that contains curated data related to human small molecule metabolism and transport.
```

File: README.md [hunk 4]
```diff
@@ -51,11 +54,20 @@ Contributions are very welcome - please follow the [guidelines](CONTRIBUTING.md)
...
+- [The Human Metabolome Database (HMDB)](https://hmdb.ca/):  is a freely available electronic database containing detailed information about small molecule metabolites found in the human body.
```
Comment (kjappelbaum) [inline README.md:66]:
```suggestion
- [The Human Metabolome Database (HMDB)](https://hmdb.ca/): is a freely available electronic database containing detailed information about small molecule metabolites found in the human body.
```

File: README.md [hunk 5]
```diff
@@ -51,11 +54,20 @@ Contributions are very welcome - please follow the [guidelines](CONTRIBUTING.md)
...
+- [KEGG PATHWAY Database(KEGG)](https://www.genome.jp/kegg/kegg2.html): a database resource for understanding high-level functions and utilities of the biological system, such as the cell, the organism and the ecosystem, from molecular-level information, especially large-scale molecular datasets generated by genome sequencing and other high-throughput experimental technologies.
```
Comment (kjappelbaum) [inline README.md:67]: Do you know if there is a way to download this data in a machine-readable form? I clicked on a few links and saw a lot of images

File: README.md [hunk 6]
```diff
@@ -51,11 +54,20 @@ Contributions are very welcome - please follow the [guidelines](CONTRIBUTING.md)
...
+- [MetXBioDB Metabolite Biotransformations](https://zenodo.org/record/4247792#.Y_uK3NJBypN): a comprehensive collection of biotransformation reactions and metabolite information from the BioTransformer database. It includes the transformation and metabolism of metabolites.
+- [QSAR datasets - Meta-QSAR (phase I & II)](https://data.mendeley.com/datasets/spwgrcnjdg/1):  Data used in Olier et al. Meta-QSAR: a large-scale application of meta-learning to drug design and discovery.
```
Comment (kjappelbaum) [inline README.md:69]:
```suggestion
- [QSAR datasets - Meta-QSAR (phase I & II)](https://data.mendeley.com/datasets/spwgrcnjdg/1):  Data (extracted from ChEMBL) used in Olier et al. Meta-QSAR: a large-scale application of meta-learning to drug design and discovery.
```

File: README.md [hunk 7]
```diff
@@ -51,11 +54,20 @@ Contributions are very welcome - please follow the [guidelines](CONTRIBUTING.md)
...
+- [EPA CompTox](https://comptox.epa.gov/dashboard): is a widely used resource for chemistry, toxicity, and exposure information for hundreds of thousands of chemicals includes, but is not limited to, chemical properties, environmental fate and transport, hazard, in vitro to in vivo extrapolation (IVIVE), exposure, bioactivity(each data has it's own licence).
```
Comment (kjappelbaum) [inline README.md:70]:
```suggestion
- [EPA CompTox](https://comptox.epa.gov/dashboard): is a widely used resource for chemistry, toxicity, and exposure information for hundreds of thousands of chemicals including, but not limited to, chemical properties, environmental fate, and transport, hazard, in vitro to in vivo extrapolation (IVIVE), exposure, bioactivity (each data has its license).
```

are you sure about the license, I found this https://epa.figshare.com/articles/dataset/Chemistry_Dashboard_Data_DSSTox_SDF/5491516 here which is CC0?

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
I'm really impressed, I wasn't aware of all the things you added!

I left some suggestions for changes, let me know if you want to discuss them.

File: README.md (hunk 1 above)
Comment (kjappelbaum) [reply README.md:29]: and I guess the raw data is here https://sourceforge.net/projects/nmrshiftdb2/files/data/. Maybe this is also worth linking?

Comment (phalem) [reply README.md:29]: some A letter in the last make link doesn't work. Try to edit link to be like this: https://nmrshiftdb.nmr.uni-koeln.de/nmrshiftdb
While some individuals struggle with Sourceforge, enter raw data on the final line([with raw data here](https://sourceforge.net/projects/nmrshiftdb2/files/data/)).

Comment (kjappelbaum) [reply README.md:29]: >some A letter in the last make link doesn't work. Try to edit link to be like this: [nmrshiftdb.nmr.uni-koeln.de/nmrshiftdb](https://nmrshiftdb.nmr.uni-koeln.de/nmrshiftdb)

thanks for spotting!

File: README.md (hunk 5 above)
Comment (phalem) [reply README.md:67]: Yeah metabolism data have even problem in readability. We will face a challenge in dealing with this data, best things to do is to make it available until we find away to deal with or use it in another way. It is Not worth mentioning.

File: README.md (hunk 7 above)
Comment (phalem) [reply README.md:70]: I found this on about page, so I write this line.
```
Disclaimer
This resource is a compilation of information sourced from many sites, databases and sources, including U.S. Federal and state sources and international bodies. The data are not reviewed by USEPA - the user must apply judgment in use of the information. You should consult the original scientific paper or data source if possible to ensure an understanding of the context of the data contained in the Dashboard.
```

**Comment by kjappelbaum** [comment]
Thanks!

**Pull request merged**